src/modules/Login/action.js

```diff
@@ -59,7 +48,7 @@ export const initializeAccount = (account: EdgeAccount, touchIdInfo: Object) =>
     walletId: '',
     currencyCode: '',
     currencyPlugins: [],
-    otpInfo: { enabled: account.otpKey != null, otpKey: account.otpKey, otpResetPending },
+    otpInfo: { enabled: account.otpKey != null, otpKey: account.otpKey, otpResetPending: false },
```

Comment:
We should probably get rid of `otpInfo.otpResetPending`, but that's scary without Flow types.

Reply:
Hence why i left it there. `false` is always safe for now